packages/controllers/src/snaps/SnapController.ts

```diff
@@ -839,18 +840,18 @@ export class SnapController extends BaseController<
     );
 
     this.messagingSystem.registerActionHandler(
-      `${controllerName}:getAllSnaps`,
+      `${controllerName}:getAll`,
       (...args) => this.getAllSnaps(...args),
     );
 
     this.messagingSystem.registerActionHandler(
-      `${controllerName}:onSessionOpen`,
-      (...args) => this.onSessionOpen(...args),
+      `${controllerName}:incrementActiveReferences`,
+      (...args) => this.incrementActiveReferences(...args),
```

Comment:
Hot tip:
if you use
```typescript
this.messagingSystem.registerActionHandler(
      `${controllerName}:incrementActiveReferences`,
      this.incrementActiveReferences.bind(this)
);
```

The coverage won't complain about untested function.

Reply:
They are already tested 😉